app/views/school_groups/current_scores.html.erb

```diff
@@ -0,0 +1,3 @@
+<% content_for :page_title, @school_group.name %>
```

Comment:
The page titles should also reflect the action, e.g. " XYZ group | Current Scores"